app/gitcoinbot/models.py

```diff
@@ -0,0 +1,6 @@
+from django.db import models
+
+
+class GitcoinBotResponses(models.Model):
```

Comment:
Can you add a docstring for this module and class that adheres to PEP-257/PEP-258 here:

```python
# -*- coding: utf-8 -*-
"""Define Gitcoin Bot specific models."""
from django.db import models


class GitcoinBotResponses(models.Model):
    """Define the Gitcoin Bot response model for recording bot request data."""

    request = models.CharField(max_length=500, db_index=True, unique=True)
    response = models.CharField(max_length=500)
```